packages/suite/src/components/suite/Labeling/components/Metadata/index.tsx

```diff
@@ -218,8 +222,7 @@ const MetadataLabeling = (props: Props) => {
                 props.defaultVisibleValue
             )}
 
-            {((labelingAvailable && !props.payload.value) ||
-                (!labelingAvailable && device?.connected)) && (
+            {!labelingImpossible && !props.payload.value && (
```

Comment:
Nitpicking a bit here but double negatives make conditions harder to read. `labelingImpossible` is used in one place and could be rewritten to `labelingPossible` (with the appropriate condition) and be checked if the value is true rather than false. Not sure if I explained myself well but the gist of it is checking the impossible value is false is harder to read than checking the possible value is true.

Reply:
yeah, but it would make definition of labelingPossible harder to read. I would either have to change the conditions to define all cases when it is possible. Generally it is possible when I already have keys (enabled), or do not have them but device is connected which means I can get them. This looks quite OK I think!

const labelingPossible = device?.metadata.status === 'enabled' || device?.connected;

or negate the expression 
```javascript
const labelingPossible = !(device?.metadata.status !== 'enabled' && !device?.connected);
```
Which I don't like at all :D

Reply:
If the condition is too complicated to rewrite (ie: easier to write as a negative), then I guess it makes sense to leave it as is rather than going with the second example (negating the same condition :vomiting_face: ). The first example looks fine if it works for the same cases as the original one.

Reply:
the first one seems to be the best for me now.